project2/styles.css

```
@@ -0,0 +1,188 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    color: #686868;
+    font-size: 18px;
+}
+
+section {
+    background-color: #faf4e3;
+    max-width: 640px;
+    margin-right: auto;
+    margin-left: auto;
+}
+
+header {
+    background-image: url(images/shrimps_2.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 300px;
+    border-radius: 5px;
+
+}
+
+.title {
+    display: block;
+    padding-left: 30px;
+    padding-top: 20px;
+}
+
+
+div {
+    height: 258px;
+    padding-left: 10px;
+}
+
+h2 {
+    text-align: left;
+    font-size: 42px;
+    color: #223b12;
+    font-family: "Notos Sans JP", sans-serif;
+    font-weight: 700;
+    margin-bottom: 10px;
+}
+
+.subhead {
+    text-align: left;
+    padding-left: 30px;
+}
+
+h3 {
+    text-align: right;
+    font-size: 22px;
+    font-family: "Notos Sans JP", sans-serif;
+    font-weight: 700;
+    padding-right: 30px;
+}
+
+h3:hover {
+    color: #223b12;
+    transition: 400ms;
+    text-decoration: underline;
+}
+
+h4 {
+    text-align: left;
+    font-size: 22px;
+    font-family: "Notos Sans JP", sans-serif;
+    font-weight: 700;
+    margin-top: 0px;
+
+}
+
+p {
+    text-align: left;
+    margin-right: 5%;
+    font-family: "Merriweather", serif;
+    font-weight: 400;
+}
+
+ul {
+    text-align: left;
+    column-count: 2;
+    column-gap: 35px;
+    font-family: "Merriweather", serif;
+    font-weight: 700;
+    line-height: 2.5;
+    font-size: 13px;
+    padding-left: 0px;
+}
+
+.button-reset {
+    background: #223b12;
+    border-radius: 10px;
+    color: #faf4e3;
+    font-size: 20px;
+    padding: 10px;
+    text-align: center;
+}
+
+.secondheader,
+.ingredients {
+    display: inline-block;
+    vertical-align: top;
+}
+
+.secondheader {
+    width: 25%;
+    margin-top: 13;
+}
+
+.ingredients {
+    background-color: #faf4e3;
+    border: 0px solid #686868;
+    width: 65%;
+    border-top: 1.5px solid#223b12;
+    border-radius: 0px;
+}
+
+.second {
+    padding: 30px;
+}
+
+.thirdheader,
+.copy {
+    display: inline-block;
+    vertical-align: top;
+}
+
+.third {
+    padding: 30px;
+}
+
+.thirdheader {
+    width: 25%;
+    margin-top: 0;
+    padding-top: 19px;
+
+}
+
+.copy {
+    background-color: #faf4e3;
+    border: 0px solid #686868;
+    width: 65%;
+    margin-top: 0;
+    column-count: 2;
+    column-gap: 20px;
+    border-top: 1.5px solid#223b12;
```

Missing a space here:
```suggestion
    border-top: 1.5px solid #223b12;
```